src/components/Admin/Preview/Viewer.js

```diff
@@ -0,0 +1,100 @@
+import React, {Component} from 'react';
+import firebase from 'firebase/app';
+import 'firebase/firestore';
+import ItemSelector from '../ItemSelector';
+import ImagePreview from './ImagePreview';
+import getStore from '../../../utils/honestyStore.js';
+
+export default class Viewer extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			item: 'all',
+			images: []
+		};
+
+		this.items = {
+			all: {name: 'All Items', id: 'all'},
+			unknown: {name: 'Unknown Item', id: 'unknown'}
+		};
+	}
+
+	componentDidMount() {
+		firebase.initializeApp({
+			apiKey: 'AIzaSyBVuVNKx-rx2ON0RxbfGfbGPpiymbMrxj8',
+			authDomain: 'honesty-store-kiosk.firebaseapp.com',
+			projectId: 'honesty-store-kiosk'
+		});
+		this.db = firebase.firestore();
+		this.db.settings({timestampsInSnapshots: true});
+
+		getStore().then(store => {
+			Object.assign(this.items, store);
```

Comment:
With this in state, we could use [object spread](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) to create a new object instead of mutating directly (just like we could've done `this.items = { ...this.items, ...store }`

```
this.setState(prevState => ({
  items: {
    ...prevState.items,
    ...store, // ideally renamed items (and getItems)
  },
}))
```